src/paths/simple_paths.c

```diff
@@ -152,10 +148,7 @@ igraph_error_t igraph_get_all_simple_paths(const igraph_t *graph,
             VECTOR(nptr)[up] = 0;
         }
 
-        iteration++;
-        if (iteration >= 10000) {
-            iteration = 0;
-        }
+        IGRAPH_ALLOW_INTERRUPTION_LIMITED(iter, 1 << 13);
```

Comment:
I guess `1 << 13` is okay but the original code used `10000` so I wonder whether this is a typo.